tests/functional/config.py

```diff
@@ -78,3 +82,15 @@
 
 """ TT_PORTS_POOL_SIZE- Ports pool size"""
 ports_pool_size = get_int("TT_PORTS_POOL_SIZE", 5000)
+
+""" TT_DEFAULT_INFER_TIMEOUT - Timeout for CPU target device"""
+default_infer_timeout = get_int("TT_DEFAULT_INFER_TIMEOUT", 10)
+
+""" TT_DEFAULT_GPU_INFER_TIMEOUT - Timeout for CPU target device"""
+default_gpu_infer_timeout = get_int("TT_DEFAULT_GPU_INFER_TIMEOUT", 10*default_infer_timeout)
+
+""" INFER TIMEOUT """
+if target_device == "GPU":
+    infer_timeout = default_gpu_infer_timeout
+else:
```

Comment:
NCS and HDDL devices are slower than CPU

Reply:
NCS and HDDL will be changed in following tasks and PR.